packages/back-end/src/routers/event-webhooks/event-webhooks.controller.ts

```diff
@@ -372,46 +370,11 @@ export const createTestEventWebHook = async (
 ) => {
   const context = getContextFromReq(req);
 
-  if (!context.permissions.canCreateEventWebhook()) {
-    context.permissions.throwPermissionError();
-  }
-  const {
-    org: { id: organizationId },
-  } = context;
   const { webhookId } = req.body;
 
-  const webhook = await EventWebHook.getEventWebHookById(
-    webhookId,
-    organizationId
-  );
-
-  if (!webhook) throw new Error(`Cannot find webhook with id ${webhookId}`);
-
-  const payload: WebhookTestEvent = {
-    event: "webhook.test",
-    object: "webhook",
-    data: { webhookId },
-    user: req.userId
-      ? {
-          type: "dashboard",
-          id: req.userId,
-          email: req.email,
-          name: req.name || "",
-        }
-      : null,
-    projects: [],
-    tags: [],
-    environments: [],
-    containsSecrets: false,
-  };
-
-  const emittedEvent = await createEvent(organizationId, payload);
-
-  if (!emittedEvent) throw new Error("Error while creating event!");
-
-  new EventNotifier(emittedEvent.id).perform();
+  await sendEventWebhookTestEvent(context, webhookId);
 
-  return res.json({ eventId: emittedEvent.id });
+  return res.status(200);
```

Comment:
Should update the `apiCall` return type in `EventWebHookDetail.tsx` to reflect that `eventId` is no longer being returned.